readthedocs/proxito/views/hosting.py

```diff
@@ -77,7 +77,11 @@ def _resolve_resources(self):
             project = unresolved_url.project
             version = unresolved_url.version
             filename = unresolved_url.filename
-            build = version.builds.last()
+            # TODO build listing is disabled for now as it caused a big jump in
+            # query time, consider adding it back in:
+            # https://github.com/readthedocs/readthedocs.org/issues/10830
+            # build = version.builds.last()
```

Comment:
I just realized that this query should eventually be:

```suggestion
            build = version.builds.filter(success=True, state=BUILD_STATE_FINISHED).last()
```